pkg/debug/start_debug.go

```diff
@@ -122,11 +121,13 @@ func setDeploymentScale(context *k8sutil.Context, clusterNamespace, deploymentNa
 	return nil
 }
 
-func verifyDeploymentExists(context *k8sutil.Context, clusterNamespace, deploymentName string) (*appsv1.Deployment, error) {
+func VerifyDeploymentExists(context *k8sutil.Context, clusterNamespace, deploymentName string) (*appsv1.Deployment, error) {
+	fmt.Printf("waiting for deployment %s to be running\n", deploymentName)
```

Comment:
It doesn't look like we're waiting for the deployment to be running? We're just retrieving it

Reply:
correct, update the name